analytics/src/main/java/com/thejawnpaul/gptinvestor/analytics/composite/CompositeLogger.kt

```diff
@@ -0,0 +1,68 @@
+package com.thejawnpaul.gptinvestor.analytics.composite
+
+import com.thejawnpaul.gptinvestor.analytics.Analytics
+import com.thejawnpaul.gptinvestor.analytics.di.FirebaseAnalytics
+import com.thejawnpaul.gptinvestor.analytics.di.MixpanelAnalytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CompositeLogger @Inject constructor(private val loggers: Map<Analytics.Provider, Analytics>) :
```

Comment:
Why not inject firebase n mix panel instances here instead of wrapping them in a class